src/NewTools-Playground/StPlaygroundBindingAdd.class.st

```diff
@@ -0,0 +1,34 @@
+Class {
+	#name : 'StPlaygroundBindingAdd',
+	#superclass : 'StPlaygroundBindingPresenter',
+	#category : 'NewTools-Playground-View',
+	#package : 'NewTools-Playground',
+	#tag : 'View'
+}
+
+{ #category : 'actions' }
+StPlaygroundBindingAdd >> buttonAction [
+	|test|	
+		
+		bindingName text isEmpty ifTrue: [ self error: 'I need a string to by identify' . ^ nil ].
+		bindingValue text isEmpty ifTrue: [ self error: 'I need a value' . ^ nil  ].
+
+
+		test := bindingValue evaluate: bindingValue text onCompileError: [ ^ self ] onError: [ :e | ^ self ]. 		"wait more instruction about this possibility
+		test ifNil: [ self error:'test'. ^ nil ]."
+		
+		
+		interactionModel addBindingNamed: bindingName text with: test.
+		parent updatePresenterKeepingSelection.
+		self delete.
+]
+
+{ #category : 'accessing' }
+StPlaygroundBindingAdd >> nameButton [ 
+	^ 'add'
```

Comment:
I did not checkout your PR in an image so I don't know it it's automatically captialized, but if it is not, it should be.